### PR TITLE
[SYCLomatic][DOC][NFC] Merge GetStartedGuide.md into README.md.

### PR DESCRIPTION
**Pull request by zhimingwang36**

### CODE REVIEW

**Review by YuriPlyakhin — CHANGES_REQUESTED** [review]
How about adding table of contents, since the readme is quite big now?

File: README.md [hunk 1]
```diff
@@ -21,6 +21,118 @@ in daily releases. None of the branches in the project are stable or rigorously
...
+* `python 3` - [Download](https://www.python.org/downloads/)
+* `ninja` -
+[Download](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages)
+* C++ compiler
+  * Linux: `GCC` version 7.1.0 or later (including libstdc++) -
+    [Download](https://gcc.gnu.org/install/)
+  * Windows: `Visual Studio` version 15.7 preview 4 or later -
+    [Download](https://visualstudio.microsoft.com/downloads/)
+
+#### Create SYCLomatic workspace
```
Comment (YuriPlyakhin) [inline README.md]:
```suggestion
### Create SYCLomatic workspace
```

File: README.md [hunk 2]
```diff
@@ -21,6 +21,118 @@ in daily releases. None of the branches in the project are stable or rigorously
...
+
+```bat
+cd %SYCLOMATIC_HOME%
+mkdir build
+cd build
+cmake -G Ninja -DCMAKE_INSTALL_PREFIX=%PATH_TO_C2S_INSTALL_FOLDER%  -DCMAKE_BUILD_TYPE=Release  -DLLVM_ENABLE_PROJECTS="clang"  -DLLVM_TARGETS_TO_BUILD="X86;NVPTX" ..\SYCLomatic\llvm
+ninja install-c2s
+```
+
+#### Deployment
```
Comment (YuriPlyakhin) [inline README.md]:
```suggestion
### Deployment
```

File: README.md [hunk 3]
```diff
@@ -21,6 +21,118 @@ in daily releases. None of the branches in the project are stable or rigorously
...
+dpct is an alias command for c2s.
+
+## Known Issues and Limitations
+
+* SYCL\* 2020 support work is in progress.
+* 32-bit host/target is not supported.
+
 
 ## Useful Links
 * [SYCLomatic getting started](GetStartedGuide.md)
```
Comment (YuriPlyakhin) [inline README.md]:
```suggestion
```

**Review by YuriPlyakhin — CHANGES_REQUESTED** [review]

**Comment by zhimingwang36** [comment]
> How about adding table of contents, since the readme is quite big now?

done.

**Review by ziranzha — APPROVED** [review]
LGTM

**Pull request merged**